README.org

```diff
@@ -69,6 +65,22 @@ There is an option to have your user credentials (email address and password) sa
 The default is not to do this because if not properly configured it would save these unencrypted which is not a good default to have.
 Customize the variable =mastodon-auth-source-file= if you want to enable this feature.
 
+*** 2 Factor Auth
+**** Overview
+After setting up =mastodon-instance-url= set the following in your
+=.emacs= or =customize=:
+
+1. =mastodon-auth-mechanism=
+
+**** Set up
+#+BEGIN_SRC emacs-lisp
+  (setq mastodon-auth-mechanism "oauth2")
+#+END_SRC
+
+The OAuth2 library uses =plstore= to securely store the credentials.
+However, =mastodon-oauth2-client-secret= needs to be stored in a safe
+place - encryption is recommended.
```

Comment:
That isn't very actionable advice.  Where is that stored and how does the user ensure it is encrypted?

Reply:
Updated, how is it now?